java/src/org/openqa/selenium/firefox/FirefoxOptions.java

```diff
@@ -66,6 +66,7 @@ public FirefoxOptions() {
     // will enable it.
     // https://fxdx.dev/deprecating-cdp-support-in-firefox-embracing-the-future-with-webdriver-bidi/.
     addPreference("remote.active-protocols", 3);
+    enableBiDi();
```

Comment:
Would it make sense to give the user a choice to opt-out in case they really don't want to see BiDi enabled?